src/components/Hub/HubImagesList.tsx

```diff
@@ -12,28 +11,118 @@ import ImageCard from "./ImageCard"
 import HubFilters, { getSelectedFilters } from "./HubFilters"
 import SpinningLoader from "../Common/SpinningLoader"
 import { ExpandingSearchbar } from "../Common/ExpandingSearchbar"
-import { FilterCategory,  FilterParams } from "../../redux/hub/hub.types"
+import {
+  FilterCategory,
+  FilterParams,
+  HubImage,
+} from "../../redux/hub/hub.types"
 import styled from "@emotion/styled"
+import { Grid } from "@material-ui/core"
+import HubSortDropdown from "./HubSortDropdown"
+import { sortOptions } from "../../redux/hub/hub.constants"
 
-const SearchContainer = styled(Row)`
-  flex-direction: row-reverse;
-  padding: 1rem;
-`
+const sortHubImages = (images: HubImage[], selectedSortOption: string) => {
+  switch (selectedSortOption) {
+    case "name increasing":
+      return images.sort((imageA, imageB) =>
+        imageA.name < imageB.name ? -1 : imageA.name > imageB.name ? 1 : 0
+      )
+    case "name decreasing":
+      return images.sort((imageA, imageB) =>
+        imageA.name > imageB.name ? -1 : imageA.name < imageB.name ? 1 : 0
+      )
+    case "author increasing":
+      return images.sort((imageA, imageB) =>
+        imageA.author < imageB.author
+          ? -1
+          : imageA.author > imageB.author
+          ? 1
+          : 0
+      )
+    case "author decreasing":
+      return images.sort((imageA, imageB) =>
+        imageA.author > imageB.author
+          ? -1
+          : imageA.author < imageB.author
+          ? 1
+          : 0
+      )
+  }
+}
+
+const HubLibraryContainer: React.FC = ({ children }) => {
+  const HubLibraryContainerStyled = styled(Grid)`
+    background-color: ${(props) => props.theme.palette.background.default};
+    border: 1px solid ${(props) => props.theme.palette.grey[300]};
+  `
+  return (
+    <HubLibraryContainerStyled container>{children}</HubLibraryContainerStyled>
+  )
+}
+
+const HubLibraryHeader: React.FC = () => {
+  const HubLibraryHeaderStyled = styled(Grid)`
+    padding: 16px;
+    border-bottom: 1px solid ${(props) => props.theme.palette.grey[300]};
+  `
+  return (
+    <HubLibraryHeaderStyled item xs={12}>
```

Comment:
I think we should reduce the use of inline styles as much as we can

Reply:
It's not styles - it's props for the Grid component.